src/clj/tasks/load_test.clj

```diff
@@ -0,0 +1,121 @@
+(ns tasks.load-test
+  "Load test for large numbers of websockets"
+  (:require [clojure.java.io :as io]
+            [clojure.string :as string]
+            [web.db :refer [db] :as webdb]
+            [clj-async-profiler.core :as prof]
+            [gniazdo.core :as ws]
+            [clj-uuid :as uuid]
+            [org.httpkit.client :as http]
+            [web.game :refer :all]
+            [crypto.password.bcrypt :as password]
+            [monger.collection :as mc]
+            [monger.result :refer [acknowledged?]]
+            [monger.operators :refer :all]
+            [web.core :refer :all]
+            [game.core-test :refer :all]
+            [game.macros-test :refer :all]
+            [web.lobby :refer :all]))
+
+(defn safe-println [& more]
+  (.write *out* (str (clojure.string/join " " more) "\n")))
+
+(defn add-test-users [maxUsers]
+  (webdb/connect)
+  (when (not (mc/find-one-as-map db "users" {:username "TestCorp"}))
+    (mc/insert db "users"
+                    {:username         "TestCorp"
+                      :email            "TestCorp@mailinator.com"
```

Comment:
Looks like each subsequent pair is indented 1 extra. And you should separate the keys and values by a single space.